test/parallel/test-buffer.js

```diff
@@ -1476,3 +1476,26 @@ assert.equal(SlowBuffer.prototype.offset, undefined);
   // Check pool offset after that by trying to write string into the pool.
   assert.doesNotThrow(() => Buffer.from('abc'));
 }
+
+
+// Test failed or zero-sized Buffer allocations not affecting typed arrays
+{
+  const zeroArray = new Uint32Array(10).fill(0);
```

Comment:
Sure. But

> a change where this and following typed arrays become filled with equivalent garbage 

the two allocations would need to be filled with exactly the same garbage, for several allocations. Though I see what you're getting at.